lib/assets/javascripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "turbograft",
+  "version": "1.0.0",
+  "description": "Turbograft extends [Turbolinks](https://github.com/rails/turbolinks), allowing you to perform partial page refreshes.",
+  "scripts": {
+    "test": "dev test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/turbograft.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Shopify/turbograft/issues"
+  },
+  "homepage": "https://github.com/Shopify/turbograft#readme",
+  "dependencies": {
+    "promise-polyfill": "6.0.1"
```

Comment:
I don't really see how it makes sense to reimplement something so that we don't have to include a reimplementation of something. Including it as a dev dependency makes sense I guess maybe possibly? If we are leaving this as an open source project we at least need to then mention somewhere what it depends on if we aren't packaging it.